Scripts/package.py

```diff
@@ -215,6 +218,12 @@ def discover(args):
 
         for release_js in js["assets"]:
             release = Release(release_js, args.version, args.out)
+            # Copying assets of osx and chnaging value of "name" to include "arm64" as architecture and then add it to same list of releases. 
+            if release.os_name == "osx":
+                tmp_release_js = release_js
+                tmp_release_js.update({'name': 'z3-4.8.5-arm64-osx-11.0.zip'})
+                tmp_release = Release(tmp_release_js, args.version, args.out)
+                yield tmp_release
```

Comment:
I don't get it. Does this release something with arm64 in the name but with the same assets as x64 ?

Reply:
The whole release process is based on available Z3 versions, even the publish process.  dotnet publish command is called by run_publish function where value passed to choose architecture is self.platform:
https://github.com/prvshah51/dafny/blob/a38fcfd8ab9d69e40adbef0c784c8aaf7e57a642/Scripts/package.py#L142
 which is determined from the `name` value of these dictionaries:
https://github.com/prvshah51/dafny/blob/a38fcfd8ab9d69e40adbef0c784c8aaf7e57a642/Scripts/package.py#L90

Reply:
Okay. So will this change will make any `osx` release, in additional to doing an x64 release, also do an arm64 release, both using `z3-4.8.5-x64` ?

Reply:
Yes, This change adds one more release data to the list of releases which will do an `arm64-osx` release and both will be using `z3-4.8.5-x64`.